src/Kdyby/Translation/Translator.php

```diff
@@ -419,6 +437,20 @@ private function extractMessageDomain($message)
 
 
 	/**
+	 * @param string $message
+	 */
+	protected function logMissingTranslation($message)
+	{
+		if ($this->psrLogger) {
+			$this->psrLogger->notice('Missing translation', [
+				'message' => $message,
+			]);
+		}
+	}
```

Comment:
I'd probably move the panel-related code into this method as well. I mean this part.

```
if ($this->panel) {
	$this->panel->markUntranslated($message);
}
```